pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -1099,6 +1141,29 @@ func (a *ApplicationBackupController) backupVolumes(backup *stork_api.Applicatio
 			}
 		}
 	}
+	// append skipped volumes
+	backup.Status.Volumes = append(backup.Status.Volumes, skipVolInfo...)
+	// add for fail and partial success
+	if len(backup.Status.Volumes) == len(skipVolInfo) {
+		// This case signifies that none of the volumes are successfully backed up
+		// hence marking it as failed
+		partialFailed = true
```

Comment:
Actually if the  len(backup.Status.Volumes) == len(skipVolInfo) condition satisfy, it is full failure or complete failure case rt? Actually, now skipped is also called as failure, Do we really need two variable partialFailed or partialsuccess? We can just have only failed volume count and updated it when ever you see a failed volume status and at end if the failedcount and len of backup.Status.Volumes is same, we can call it as Failed and it Failecount is zero, we can call it as success. Otherwise we can called it as partial success.